image.go

```diff
@@ -362,6 +472,87 @@ func (img *imageHDU) Write(data interface{}) error {
 	return err
 }
 
+// ToImage returns a pure Go image.Image instance from the FITS image.
```

Comment:
please call it `Image` (as per the `fitsio.Image` interface) and replace "a pure Go image.Image instance" with just "an image.Image value"
(`Go` doesn't have classes nor "instances" of classes)